EIPS/eip-615.md

```diff
@@ -1,27 +1,27 @@
 ---
 eip: 615
 title: Subroutines and Static Jumps for the EVM
-status: Draft
+status: Last Call
 type: Standards Track
 category: Core
 author: Greg Colvin <greg@colvin.org>, Brooklyn Zelenka (@expede) , Paweł Bylica (@chfast), Christian Reitwiessner(@chriseth)
-discussions-to: https://ethereum-magicians.org/t/eip-615-subroutines-and-static-jumps-for-the-evm/2728
+discussions-to: https://ethereum-magicians.org/t/eip-615-subroutines-and-static-jumps-for-the-evm-last-call/3472
```

Comment:
Why change the link ?

Reply:
The original link is 150 comments long, going back to February, so it was getting really hard to follow.

Reply:
It's allowed, but difficult for readers to find the whole conversation from start to finished. If somebody in the future wants to understand the reasons why certain decisions were made, they'd need to go to the new last-call discussion and from there find the original discussion.

Reply:
Understood, thus the cross-links.  Not only was the thread already much longer than for any other EIP, but it ended with days of confusing discussion that were really about versioning proposals.  Most of the substantial decisions show up in two threads of earlier discussion from the proposal's years as Issues, also cross-linked, including work with Sydney on a formal specification of the proposal.  A lot of history.  Not, I think, too hard to follow, but more than I'd expect most reviewers to have read.  So it just seemed better to make a clear start to this phase of review.